visualization/app/codeCharta/codeCharta.service.spec.ts

```diff
@@ -2,16 +2,20 @@ import "./codeCharta.module"
 import { CodeChartaService } from "./codeCharta.service"
 import { getService, instantiateModule } from "../../mocks/ng.mockhelper"
 import { TEST_FILE_CONTENT } from "./util/dataMocks"
-import { CCFile, BlacklistType, NodeType } from "./codeCharta.model"
+import { AttributeTypes, AttributeTypeValue, BlacklistType, CCFile, NodeType } from "./codeCharta.model"
 import _ from "lodash"
 import { StoreService } from "./state/store.service"
 import { resetFiles } from "./state/store/files/files.actions"
+import { ExportBlacklistType, ExportCCFile } from "./codeCharta.api.model"
 import { getCCFiles, isSingleState } from "./model/files/files.helper"
+import { DialogService } from "./ui/dialog/dialog.service"
 
 describe("codeChartaService", () => {
 	let codeChartaService: CodeChartaService
 	let storeService: StoreService
-	let validFileContent
+	let dialogService: DialogService
+	let validFileContent: ExportCCFile
+	const fileName: string = "someFileName"
```

Comment:
Note: it is not required to define the type in situations like these. The TypeScript compiler infers the type right away.

Reply:
That is very true. Enabling this rule will solve this issue once and for all. I'll adress this in another PR though.

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-inferrable-types.md